package.json

```diff
@@ -1,54 +1,29 @@
 {
   "name": "hubot",
```

Comment:
We should update the name of this package too as part of forking this, and the version below.